app.config.js

```diff
@@ -37,6 +38,7 @@ const config = {
           "To optimize app launch times and resource utilization based on system boot information.",
         UIBackgroundModes: ["audio"],
       },
+      // associatedDomains: ["applinks:app.teritori.com"], FIXME: when adding this ios is not build but it is needed for deep linking
```

Comment:
so deep linking doesn't work?

Reply:
When it is enabled ios build failed. so, yeah it didn't work.